static/app/views/organizationGroupDetails/groupTags.tsx

```diff
@@ -2,158 +2,124 @@ import * as React from 'react';
-        this.setState({
-          tagList: data,
-          error: false,
-          loading: false,
-        });
-      },
-      error: () => {
-        this.setState({
-          error: true,
-          loading: false,
-        });
-      },
-    });
-  };
-
-  getTagsDocsUrl() {
-    return 'https://docs.sentry.io/platform-redirect/?next=/enriching-events/tags';
-  }
-
-  render() {
+  renderTags() {
     const {baseUrl} = this.props;
+    const {tagList} = this.state;
 
-    let children: React.ReactNode[] = [];
-
-    if (this.state.loading) {
-      return <LoadingIndicator />;
-    } else if (this.state.error) {
-      return <LoadingError onRetry={this.fetchData} />;
-    }
+    const alphabeticalTags = (tagList ?? []).sort((a, b) => a.key.localeCompare(b.key));
 
-    if (this.state.tagList) {
-      children = this.state.tagList.map((tag, tagIdx) => {
-        const valueChildren = tag.topValues.map((tagValue, tagValueIdx) => {
-          let label: React.ReactNode = null;
-          const pct = percent(tagValue.count, tag.totalValues);
-          const query = tagValue.query || `${tag.key}:"${tagValue.value}"`;
-
-          switch (tag.key) {
-            case 'release':
-              label = <Version version={tagValue.name} anchor={false} />;
-              break;
-            default:
-              label = <DeviceName value={tagValue.name} />;
-          }
-
-          return (
-            <li key={tagValueIdx} data-test-id={tag.key}>
-              <TagBarGlobalSelectionLink
-                to={{
-                  pathname: `${baseUrl}events/`,
-                  query: {query},
-                }}
-              >
-                <TagBarBackground style={{width: pct + '%'}} />
-                <TagBarLabel>{label}</TagBarLabel>
-                <TagBarCount>
-                  <Count value={tagValue.count} />
-                </TagBarCount>
-              </TagBarGlobalSelectionLink>
-            </li>
-          );
-        });
-
-        return (
+    return (
+      <Container>
+        {alphabeticalTags.map((tag, tagIdx) => (
           <TagItem key={tagIdx}>
             <Panel>
-              <PanelHeader hasButtons style={{textTransform: 'none'}}>
-                <div style={{fontSize: 16}}>{tag.key}</div>
-                <DetailsLinkWrapper>
-                  <GlobalSelectionLink
-                    className="btn btn-default btn-sm"
-                    to={`${baseUrl}tags/${tag.key}/`}
-                  >
-                    {t('More Details')}
-                  </GlobalSelectionLink>
-                </DetailsLinkWrapper>
-              </PanelHeader>
+              <StyledPanelHeader hasButtons>
+                <TagHeading>{tag.key}</TagHeading>
+                <GlobalSelectionLink
+                  className="btn btn-default btn-sm"
+                  to={`${baseUrl}tags/${tag.key}/`}
+                >
+                  {t('More Details')}
+                </GlobalSelectionLink>
```

Comment:
I think I used something like this previously:
```
<Button
    to={{
      pathname: `/organizations/${orgSlug}/something`,
      query: extractSelectionParameters(location.query),
    }}
 >
```

Reply:
thanks!!